lib/theme_check/offense.rb

```diff
@@ -144,6 +144,36 @@ def correct
         corrector = Corrector.new(template: template)
         correction.call(corrector)
       end
+    rescue => e
+      ThemeCheck.bug(<<~EOS)
+        Exception while running `Offense#correct`:
+        ```
+        #{e.class}: #{e.message}
+          #{e.backtrace.join("\n  ")}
+        ```
+
+        Template:
+        #{template.path}
+        Check:
+        #{code_name.strip}
+        Markup:
+        ```
+        #{markup}
+        ```
+        Node.Markup:
+        ```
+        #{node&.markup}
+        ```
+        Check options:
+        ```
+        #{check.options.pretty_inspect}
+        ```
+        Offense:
+        ```
+        #{JSON.pretty_generate(to_h)}
```

Comment:
Won't this repeat the info from above?

Reply:
A bit :P Maybe I should stick to just this.